collections/recommendations/collections/REC_rc-md-item-type.adoc

```diff
@@ -0,0 +1,6 @@
+[[rec_collections_rc-md-item-type]]
+[width="90%",cols="2,6a"]
+|===
+^|*Recommendation {counter:rec-id}* |*/rec/collections/rc-md-item-type* 
+^|A |If it is possible and reasonable to interact directly with the items of a collection via an accessible `/collection/{collectionId}/items` endpoint then the `itemType` key SHOULD be included in the collection object to indicate the type of the items (e.g. `feature` or `record`).
```

Comment:
I would simply say "If a resource at the path `/collection/{collectionId}/items` is supported by the API, then the `itemType` ..."

Reply:
I suggest to replace the "(e.g...)" by: ". OGC APIs will define values for types of items (e.g. OGC API features defines 'feature' and OGC API record will define 'record')"

Reply:
/collection/{collectionId}/items should be /collections/{collectionId}/items`